app-modules/form/src/Http/Controllers/FormWidgetController.php

```diff
@@ -254,4 +270,79 @@ public function store(
             'message' => 'Form submitted successfully.',
         ]);
     }
+
+    public function registerProspect(Request $request, Form $form): JsonResponse
+    {
+        $this->validate($request, [
+            'email' => 'email|required|unique:prospects,email',
+            'first_name' => 'required|max:255',
+            'last_name' => 'required|max:255',
+            'preferred' => 'required|max:255',
+            'mobile' => 'required|max:255',
+            'bitrhdate' => 'required|date',
+            'address' => 'required|max:255',
+            'address_2' => 'required|max:255',
+            'city' => 'required|max:255',
+            'state' => 'required|max:255',
+            'postal' => 'required|max:255',
+        ]);
+
+        $prospect = Prospect::query()
+            ->make([
+                'email' => $request->get('email'),
+                'first_name' => $request->get('first_name'),
+                'last_name' => $request->get('last_name'),
+                'full_name' => "{$request->get('first_name')} {$request->get('last_name')}",
+                'preferred' => $request->get('preferred') ?? null,
+                'bitrhdate' => $request->get('bitrhdate'),
+                'mobile' => $request->get('mobile'),
+                'address' => $request->get('address'),
+                'address_2' => $request->get('address_2'),
+                'city' => $request->get('city'),
+                'state' => $request->get('state'),
+                'postal' => $request->get('postal'),
```

Comment:
Once the refactor to a form request class has happened, you could refactor a lot of this to just `$request->validated()`, merging in the `full_name` yourself